packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -5627,6 +6107,25 @@ The catalog is currently not exposed via an API. While additional apps may be ad
 
 The current workaround is to manually configure the desired application via the administrator UI in a preview (sandbox) organization and view the application via [Get Application](#get-application).
 
+##### App settings
+
+Each application has a schema that defines the required and optional settings for the application. When adding an application, you must specify the required settings.
+
+The catalog is currently not exposed via an API. While additional apps may be added via the API, only the following template applications are documented:
```

Comment:
Currently, the OIN catalog isn't available through an API. While additional app integrations can be added via the API, only the following template applications are documented:

(there's no template applications listed here - are there supposed to be?)